include/mbedtls/config.h

```diff
@@ -270,6 +270,7 @@
 //#define MBEDTLS_CMAC_ALT
 //#define MBEDTLS_DES_ALT
 //#define MBEDTLS_RSA_ALT
```

Comment:
This list was almost in alphabetical order, please move `RSA_ALT` and `XTEA_ALT` to its rightful place and re-run `generate_features.pl`.

Reply:
Fixed in d8b3851.